inst/qml/LSbinaryclassification.qml

```diff
@@ -0,0 +1,157 @@
+//
+// Copyright (C) 2013-2018 University of Amsterdam
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public
+// License along with this program.  If not, see
+// <http://www.gnu.org/licenses/>.
+//
+import QtQuick 2.8
+import QtQuick.Layouts 1.3
+import JASP.Controls 1.0
+import JASP.Widgets 1.0
+import JASP.Theme 1.0
+import "../qml/qml_components" as LS
+
+Form {
+	Section
+	{
+		title: qsTr("Input")
+		columns: 1
+		RadioButtonGroup
+		{
+			name: "inputType"; id: inputType; columns: 3; title: qsTr("Input type")
+			RadioButton { name: "pointEstimates";		label: qsTr("Point estimates"); checked: true	}
+			RadioButton { name: "uncertainEstimates";	label: qsTr("Uncertain estimates")				}
+			RadioButton { name: "data";					label: qsTr("Load data and specify threshold")	}
+		}
+
+		Group
+		{
+			visible: inputType.value === "pointEstimates"
+			title: qsTr("Estimates")
+			FormulaField { name: "prevalence";	label: qsTr("Prevalence");	min: 0; max: 1; defaultValue: "0.1"	}
```

Comment:
Maybe a different `min` value for these to prevent 
[binaryClassification_2.zip](https://github.com/jasp-stats/jaspLearnBayes/files/6413598/binaryClassification_2.zip)